samples/cloud-task/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+${PWD%/*samples/*}/scripts/compileWithMaven.sh -Dspring.native.build-time-properties-checks=default-include-all -Dspring.native.build-time-properties-match-if-missing=false -Dspring.native.factories.no-actuator-metrics=true && sleep 3 && ${PWD%/*samples/*}/scripts/test.sh
```

Comment:
Could you please share more about those custom properties? I am asking because samples are expected to be able to run with the default configuration in order to keep things simple for users.

Reply:
Updated the PR to remove the custom properties.   These were added to match those in the batch sample.    Do they need to be removed from the batch sample as well?

Reply:
Thanks, for the batch sample this will be fixed as part of #203.